CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # PowerModelsDistribution.jl Change Log
 
+## LPUBF
+
+- Add constraint_mc_transformer_power_yy(pm::LPUBFDiagModel,..)
+- Add constraint_mc_transformer_power_dy(pm::LPUBFDiagModel,..)
+- Edited constraint_mc_power_losses(pm::LPUBFDiagModel..)
+- Edited constraint_constraint_mc_model_voltage_magnitude_difference(pm::LPUBFD..) to handle buses with not all 4 terminals active
+- Edited variable bounds in variable.jl to fix inconsistencies
+- Added unit tests
+
```

Comment:
Should be condensed to a single bullet in the `staged` section, e.g. 

`- Add wye-wye and delta-wye constraints to LPUBFDiagModel formulation (#289)`